### ISSUE TITLE
import-blacklist: allow package name preferences?

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [palantir/tslint#3843]
_(forking from https://github.com/Microsoft/tslint-microsoft-contrib/issues/422)_

When there are two modules that declare variables with the same name, it's easy to mistakenly import from the wrong package:

```typescript
import { action } from "mobx"; // this one?
import { action } from "mobx-safe"; // or this one?
```

A rule that lets projects map imports to their preferred package imports would help:

```json
{
    "rules": {
        "import-blacklist": [
            true,
            "rxjs",
            {
                "mobx-safe": {
                    "action": {
                        "override": ["*"],
                        "reason": ["Technical mumbo jumbo..."]
                    }
                }
            }
        ]
    }
}
```

...though this config could be misleading, as the `"rxjs"` setting tends away from that module while `"mobx-safe"`'s tends towards that one. I'm not sure if I have a preference to add to `import-blacklist` or make a new rule.

**Comment by ethanresnick** [comment]
@JoshuaKGoldberg Would my PR here address your use case? https://github.com/palantir/tslint/pull/3926

**Comment by JoshuaKGoldberg** [comment]
Looks like it!

**Comment by JoshuaKGoldberg** [comment]
Fixed by #3926 🙌
